easybuild/easyblocks/b/binutils.py

```diff
@@ -99,6 +109,34 @@ def configure_step(self):
         # complete configuration with configure_method of parent
         super(EB_binutils, self).configure_step()
 
+        if self.cfg['install_libiberty']:
+            cflags = os.getenv('CFLAGS')
+            if cflags:
+                self.cfg.update('buildopts', 'CFLAGS="$CFLAGS -fPIC"')
+            else:
+                # if $CFLAGS is not defined, make sure we retain "-g -O2",
+                # since not specifying any optimization level implies -O0...
+                self.cfg.update('buildopts', 'CFLAGS="-g -O2 -fPIC"')
+
+    def install_step(self):
+        """Install using 'make install', also install libiberty if desired."""
+        super(EB_binutils, self).install_step()
+
+        # only install libiberty files if if they're not there yet;
+        # libiberty.a is installed by default for old binutils versions
+        if self.cfg['install_libiberty']:
+            if not os.path.exists(os.path.join(self.installdir, 'include', 'libiberty.h')):
+                copy_file(os.path.join(self.cfg['start_dir'], 'include', 'libiberty.h'),
+                          os.path.join(self.installdir, 'include', 'libiberty.h'))
+
+            if not glob.glob(os.path.join(self.installdir, 'lib*', 'libiberty.a')):
+                copy_file(os.path.join(self.cfg['start_dir'], 'libiberty', 'libiberty.a'),
+                          os.path.join(self.installdir, 'lib', 'libiberty.a'))
+
+            if not os.path.exists(os.path.join(self.installdir, 'info', 'libiberty.texi')):
+                copy_file(os.path.join(self.cfg['start_dir'], 'libiberty', 'libiberty.texi'),
+                          os.path.join(self.installdir, 'info', 'libiberty.texi'))
```

Comment:
Although this has been merged already: does it really make sense to install the `.texi` file?  I thought that this is the input file for `texinfo` to generate the actual `.info` file to be installed...